src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int FIGURE_RANDOM_PARAMETR = 20;
+    private static final int DEFAULT_RADIUS = 10;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        String figureColor = colorSupplier.getRandomColor();
+        int randomNumber = random.nextInt(FIGURE_COUNT);
+
+        switch (randomNumber) {
+            case 0:
+                return createCircle(figureColor);
+            case 1:
+                return createSquare(figureColor);
+            case 2:
+                return createRightTriangle(figureColor);
+            case 3:
+                return createRectangle(figureColor);
+            case 4:
+                return createIsoscelesTrapezoid(figureColor);
+            default:
+                return new Circle(10, Color.WHITE.name());
```

Comment:
```suggestion
            case 4:
            default:
                return createIsoscelesTrapezoid(figureColor);
```
we should never exceed this number (it means there was a programmer mistake), so let's just combine default case and the last case